lib/tls.js

```diff
@@ -235,4 +235,8 @@ exports.TLSSocket = require('_tls_wrap').TLSSocket;
 exports.Server = require('_tls_wrap').Server;
 exports.createServer = require('_tls_wrap').createServer;
 exports.connect = require('_tls_wrap').connect;
-exports.createSecurePair = require('_tls_legacy').createSecurePair;
+
+exports.createSecurePair =
```

Comment:
Yeah i was considering that. Perhaps it's worthwhile checking how common it is to require that file directly